package.json

```
@@ -9,10 +9,10 @@
     "lint": "npm run lint:eslint && npm run lint:prettier",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:prettier -- --write",
     "build": "npm run build:sass && npm run build:js && npm run copy-assets",
-    "build:js": "npm run build:js:application && npm run build:js:cookies && npm run build:js:all",
+    "build:js": "yarn build:js:application && yarn build:js:analytics && yarn build:js:all",
```

```suggestion
    "build:js": "npm run build:js:application && npm run build:js:analytics && npm run build:js:all",
```
I know the yarn command is shorter but we use npm in this repo so let's not mix them up.

---

Thank you - I mixed up the repos - fixed!